2-core-rules.md

```diff
@@ -214,108 +209,97 @@ _Fleeing_ models must take a _leadership test_. If the test is passed, the model
 
-Any model that runs loses its chance to shoot during that turn. He is concentrating on running and is not prepared to fight, having sheathed or shouldered his weapons. You should declare that models are running as they move, as this will remind both players that the model is unable to shoot that turn. Running models can cast spells as normal. 
+Any model that runs loses its chance to shoot during that turn. They are concentrating on running and is not prepared to fight, having sheathed or shouldered their weapons. You should declare that models are running as they move, as this will remind both players that the model is unable to shoot that turn. Running models can cast spells as normal. 
 
 #### Charge!
 
-If you want a model to engage the enemy in hand-to-hand combat then you must make a special move called a charge. Without measuring the distance, declare that your model is charging and indicate which enemy model he is going to attack. You can charge any opposing model if you can draw an unobstructed line from your model to the target. If your warrior wants to charge an enemy model within 4" that he can't see (e.g., because it is behind a corner) but has not been declared as hidden, he must pass an Initiative test to detect it. If he fails the roll, your model may not charge this turn, but may move his normal distance, shoot and cast spells. A charge is like a running move, performed at double the Movement rate, but ends with the attacker moving by the most direct route into base-to-base contact with the enemy model. Once their bases are touching they are engaged in hand-to-hand combat. Models are also considered to be in hand-to-hand combat even when separated by a low wall or obstacle, where it is impossible for the bases to touch physically because the obstacle is in the way.
+If you want a model to engage the enemy in hand-to-hand combat then you must make a special move called a charge. Declare to your opponent that your model is charging and indicate which enemy model they are going to attack. You can charge any opposing model if your warrior has the movement to reach them and has them in sight. If your warrior wants to charge an enemy model that they can't see (e.g., because it is behind a corner or it is in _Hiding_), they must pass an Initiative test to detect it. If they fail the roll your model may not charge this turn as they tentatively roam forwards in search of their opponent. They move up to 4" towards the target, stopping no closer than 1" away from them. A charge is like a running move, performed at double the Movement rate, but ends with the attacker moving into base-to-base contact with the enemy model. Once their bases are touching they are engaged in hand-to-hand combat. Models are also considered to be in hand-to-hand combat even when separated by a low wall or obstacle, where it is impossible for the bases to touch physically because the obstacle is in the way.
```

Comment:
```suggestion
If you want a model to engage the enemy in hand-to-hand combat then you must make a special move called a charge. Declare to your opponent that your model is charging and indicate which enemy model they are going to attack. You can charge any opposing model if your warrior has the movement to reach them and has them in sight. If your warrior wants to charge an enemy model that they can't see (e.g., because it is behind a corner or it is in _Hiding_), they must pass an Initiative test to detect it. If they fail the roll your model may not charge this turn as they tentatively roam forwards in search of their opponent. They move up to 4" towards the target, stopping no closer than 1" away from them. 

A charge is like a running move, performed at double the Movement rate, but ends with the attacker moving into base-to-base contact with the enemy model. Once their bases are touching they are engaged in hand-to-hand combat. Models are also considered to be in hand-to-hand combat even when separated by a low wall or obstacle, where it is impossible for the bases to touch physically because the obstacle is in the way.
```

Reply:
I like your change!